### DISCUSSION TITLE
the use of Blender’s Grease Pencil tool, specifically regarding the use of tools such as Ctrl+T (the shape deformation tool) and the path adjustment tool for drawing

### DISCUSSION THREAD

**Discussion by Sheng991116** [chsh2/nijiGPen#30] [General]
Hello author, your plugin has been a great help for me as a beginner in learning and understanding Blender's Grease Pencil tool. Thank you very much! I wanted to share my own experience: while learning to use the Grease Pencil tool, I found two additional plugins that are very effective in exploring the practical applications of the tool. If your plugin had similar features, then Blender's Grease Pencil tool could almost replace mainstream painting software like Photoshop, and even potentially create a true 3D vector drawing workflow.

1. Pack of extended tool for Grease Pencil (open source)
https://github.com/Pullusb/greasepencil-addon
Main features: perspective transformation with Ctrl+T, managing Grease Pencil layers

2. GP draw transform (paid)
https://pullusb.gumroad.com/l/gp_draw_transform
Main features: more precise control of Grease Pencil strokes

Together with your plugin, these two plugins would create a complete set of mainstream vector drawing features, including vector drawing, path adjustment, shape adjustment, and boolean operations.

**Comment by chsh2** [comment]
Thank you for sharing your thoughts. I personally use Grease Pencil Tools frequently for its Ctrl+T box deforming, and I recently also started to pay attention to GP draw transform. I will keep tracking other Grease Pencil add-ons to ensure the synergy with mine and avoid conflicts.
Besides add-ons, I also use the tools in the Sculpt mode for path adjustment. I found them working very smoothly compared to similar tools in other vector drawing software.
